src/controllers/mainController.ts

```diff
@@ -191,9 +188,9 @@ export default class MainController implements vscode.Disposable {
                         Constants.cmdScriptSelect, async (node: TreeNodeInfo) => {
                     const uri = await this._untitledSqlDocumentService.newQuery();
                     if (!this.connectionManager.isConnected(uri.toString())) {
-                        const connectionCreds = node.connectionCredentials;
+                        let connectionCreds = node.connectionCredentials;
                         const databaseName = `${escapeCharacters(self.getDatabaseName(node))}`;
```

Comment:
Why did you change this? const just means you can't change what connectionCreds is pointing to - you're still free to modify the properties of a const.